dist/image-sequencer-ui.js

```diff
@@ -489,6 +541,13 @@ function IntermediateHtmlStepUi(_sequencer, step, options) {
       .insertAdjacentElement('afterend',
         addStepUI
       );
+      console.log(sequencer);
```

Comment:
Oops, looks like a stray log file and some commented out code here?

Reply:
Also oops i commented on the /dist/ version... 😂 anyways you get it!

Reply:
Sure! :smile: